File: Logitude/AngularModules/AngularModules/Infrastructure/DataContracts/AutomationCondition.ts


import {Injectable} from '@angular/core';

@Injectable()

export class AutomationCondition {

    public Id: string;
    public CreateDate: Date;
    public UpdateDate: Date;
    public AutomationsId: string;
    public Tenant: number;

    public ObjectFieldId: string;
    public OperatorCode: string;
    public Value: string;
    public ConditionType: string;
    public CreatedByUserId: string;

    public UpdatedByUserId: string;
    public ObjectFieldType: string;


}

File: Logitude/Simplog.Data/Helpers/AutomationCondition.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Simplog.Data.Helpers
{

  [DataContract(Namespace = "")]
    public class AutomationCondition
    {
   
        [DataMember]
        public int Tenant { get; set; }
  
        [DataMember]
        public string ObjectFieldId { get; set; }
        [DataMember]        
        public string OperatorCode { get; set; }
        [DataMember]
        public string Value { get; set; }
        [DataMember]
        public string ConditionType { get; set; }
        [DataMember]
        public DateTime? CreateDate { get; set; }
        [DataMember]
        public DateTime? UpdateDate { get; set; }
        [DataMember]
        public string CreatedByUserId { get; set; }
        [DataMember]
        public string UpdatedByUserId { get; set; }

        [DataMember]
        public string ObjectFieldType { get; set; }

 
    }
}

